doc/source/conf.py

```diff
@@ -298,8 +303,94 @@
     for page in moved_api_pages
 }
 
+
+common_imports = """\
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import numpy as np
+   from pandas import *
+   import pandas as pd
+   randn = np.random.randn
+   np.set_printoptions(precision=4, suppress=True)
+   options.display.max_rows = 15
+"""
+
+import re
+import git
+
+xpr = re.compile(r'^v(\d+)\.(\d+)\.(\d+)$')
+repo = git.Repo('..')
+tags = [x.name for x in repo.tags if xpr.match(str(x))]
+versions = sorted(zip(tags,
+                      [tuple(map(int, xpr.match(x).groups())) for x in tags]),
+                  key=lambda x: x[1])
```

Comment:
In case you don't know it, `distutils` provides a class to parse, sort... versions. You'd need to get rid of the heading `v` in `v0.23.1`. But then you can do this stuff automatically. For example:

```
>>> import distutils.version
>>> tags = ['0.22.0', '0.23.0', '0.23.1', '0.21.1', '0.21.0']
>>> versions = map(distutils.version.StrictVersion, tags)
>>> versions = sorted(versions)
>>> versions[-1].version
(0, 23, 1)
```